src/index.js

```diff
@@ -53,7 +54,13 @@ function prettify(text, formatOptions) {
 
 function eslintFix(text, eslintConfig) {
   const eslintOptions = {
+    // overrideables
+    useEslintrc: false,
```

Comment:
Yeah, I think that makes sense. Normally, I expect people to simply pass the `filePath` and we'll get the `eslintConfig` from that (see `getConfig` [below](https://github.com/kentcdodds/prettier-eslint/blob/master/src/index.js#L79)). But if people are providing some special config in addition to what ESLint can find with the `.eslintrc` then that makes sense :)